odl/trafos/wavelet.py

```diff
@@ -41,6 +41,114 @@
 _SUPPORTED_IMPL = ('pywt',)
 
 
+def coeff_size_list_axes(shape, wbasis, mode, axes):
```

Comment:
Addition: I see that this function has no `nscales` argument. You can easily add it and make `nscales` and `axes` mutually exclusive.